config/webspaces/sulu-blog.xml

```diff
@@ -15,7 +15,7 @@
     </localizations>
```

Comment:
whats the reason for adjusting this file?

Reply:
To have all the features somehow implemented in this repository. We've decided some time ago to include as much as possible here to make testing easier. In this repository it does not matter if the configuration gets a bit more complicated, because these files are not used in a real project.